libraries/AP_BoardConfig/AP_BoardConfig_CAN.h

```diff
@@ -130,7 +133,9 @@ class AP_BoardConfig_CAN {
     Interface _interfaces[MAX_NUMBER_OF_CAN_INTERFACES];
     Driver _drivers[MAX_NUMBER_OF_CAN_DRIVERS];
     uint8_t _num_drivers;
-
+    AP_Int8 _slcan_rt;
```

Comment:
see comments above on variable names and bit width

Reply:
@tridge fixed in commit https://github.com/ArduPilot/ardupilot/pull/10171/commits/79c9126477114b1309430d6e997af5f850ec8ec6